app.rb

```diff
@@ -87,7 +87,11 @@ def config
       else
         error = params[:error] || 'missing callback param: code'
 
-        erb :errors, locals: { error: error }
+        if error == 'access_denied'
+          redirect to('/')
```

Comment:
Can we render a warning banner or something? Or maybe an info banner with "you chose to exit before signing in"? Maybe add `?error=access_denied` in the URL so the index page can handle it?

Reply:
Thanks for 1910a1e! Can we update the index action to render something based on that error param too?